README-minishift.md

```diff
@@ -243,11 +260,13 @@ Now use the `kubefed2` tool to register (*join*) the two clusters:
 kubefed2 join cluster1 \
             --host-cluster-context cluster1 \
             --add-to-registry \
-            --v=2
+            --v=2 \
+            --federation-namespace=federation-system
```

Comment:
I believe that `federation-system` is the default namespace, right? If so, I'm wondering why/if we should add this explicit flag about it here.

Reply:
You're right, it is the default namespace. I thought it helps to understand why kubefed2 resources are created in that specific namespace (even when you don't explicitly use `--federation-namespace` parameter).

I can remove the parameter and write a note explaining this before these steps, though.